redhat/frr.spec.in

```diff
@@ -779,6 +795,10 @@ sed -i 's/ -M rpki//' %{_sysconfdir}/frr/daemons
 
 
 %changelog
+* Tue Jun 21 2022 Mobashshera Rasool <mrasool@vmware.com> - %{version}
```

Comment:
This shouldn't happen here. We update changelog only for releases.

Reply:
Oh I did not know about it. I referred few commits where it was added. No problem, I will remove it. 😄

Reply:
Removed it.